src/Transform/ONNX/Decompose.cpp

```diff
@@ -84,6 +84,18 @@ DenseElementsAttr createDenseArrayAttrOrEmpty(
   }
 }
 
+Value createSequenceConstructOp(
+    PatternRewriter &rewriter, mlir::Value seq, mlir::OperandRange inputs) {
+  auto resType = seq.getType();
```

Comment:
Suggest to use the correct type (`Type`, `Location`, etc...) rather than using `auto`. The reader will know immediately what is the type of the result, while using `auto` one is left to wonder.

Reply:
fixed

Reply:
same here, I don't see it.